products/protractor.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
+iconSlug: protractor
+permalink: /protractor
+versionCommand: npm list protractor
+
+releaseDateColumn: true
+activeSupportColumn: Community Support
+extendedSupportColumn: Commercial Support
+eolColumn: false
+
+auto:
+-   npm: protractor
+
+identifiers:
+-   purl: pkg:npm/protractor
+
+releases:
+-   releaseCycle: "7.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: true
+    latest: "7.0.0"
+    latestReleaseDate: 2020-05-13
+    releaseDate: 2020-05-13
+    link: https://github.com/angular/protractor/blob/release-7.0/CHANGELOG.md#700
+
+-   releaseCycle: "6.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "6.0.0"
+    latestReleaseDate: 2190-03-23
```

Comment:
Fast catch, according to https://github.com/angular/protractor/releases/tag/6.0.0:
```suggestion
    latestReleaseDate: 2019-03-23
```
I verify all the details later...